tt_metal/hw/ckernels/wormhole_b0/metal/llk_api/llk_sfpu/ckernel_sfpu_remainder.h

```diff
@@ -57,7 +57,13 @@ inline void calculate_remainder(const uint value, const uint recip) {
             v = std::numeric_limits<float>::quiet_NaN();
         }
         v_endif;
-
+        for(int l=0; l<10; l++)
```

Comment:
Would prefer to avoid magic constants. Can you define a `constexpr auto [descriptive name here] = 10;` and use that in the `for` loop instead? How was this number determined?

Reply:
Yes I will use constexpr, and the 10 is that I got it after running multiple times, this makes all the cases passing